mysql-test/suite/galera/t/galera_sst_clone.test

```diff
@@ -0,0 +1,25 @@
+# run it with --only-combinations=enc_off
+
+--source include/big_test.inc
+--source include/galera_cluster.inc
+
+--disable_query_log
+--connection node_1
+CALL mtr.add_suppression("Failed to remove page file");
+--connection node_2
+CALL mtr.add_suppression("Failed to remove page file");
+--enable_query_log
+
+
+CALL mtr.add_suppression("Found an entry in the 'role_edges' table with unknown authorization ID '`mysql.pxc.sst.user`@`localhost`'" );
```

Comment:
No, we have to understand why it happens, not just suppress. It may be the indication of the problem that if masked now, will pop up as a bug in the future.

Reply:
Well we know the why.
When tests are executed in very fast machines, IF you do not revoke the role to the pxc user properly (with Revoke) than the cleanup is implicit and delegated to mysql.
What happens here is that the cleanup had not YET take place and the select to check the table (role) status differs from the start.
However IF the cleanup is properly done (with explicit Revoke) than no issue. 
This was also the reason to have the code:
https://github.com/Tusamarco/percona-xtradb-cluster/blob/a31f138a9a1976bedf36e89f5b96cd59c0549172/sql/wsrep_sst.cc#L1245 
`                        "REVOKE mysql.pxc.sst.role FROM 'mysql.pxc.sst.user'@'localhost';",`
in wsrep_sst.cc

Reply:
Multiple occurrences.
Please remove this suppression. Related to [THIS](https://github.com/percona/percona-xtradb-cluster/pull/1961/files#r1886504693) comment.

Reply:
done